src/Illuminate/Foundation/Providers/ArtisanServiceProvider.php

```diff
@@ -722,7 +722,7 @@ protected function registerScopeMakeCommand()
     protected function registerSeederMakeCommand()
     {
         $this->app->singleton(SeederMakeCommand::class, function ($app) {
-            return new SeederMakeCommand($app['files'], $app['composer']);
+            return new SeederMakeCommand($app['files']);
```

Comment:
`SeederMakeCommand` extends `GeneratorCommand` and doesn't use `Illuminate\Support\Composer`.